rskj-core/src/test/java/org/ethereum/rpc/exception/RskErrorResolverTest.java

```diff
@@ -80,17 +81,65 @@ public void test_resolveError_givenUnrecognizedPropertyException_nullExceptionMe
 
         // Then
         Assert.assertNotNull(result);
-        Assert.assertEquals(-32603, result.code);
+        Assert.assertEquals(-32602, result.code);
```

Comment:
Maybe use the constant here? Or is it hardcoded so that if the constant changes we have to update the test?

Reply:
This comment applies for all of the usages below